src/entrypoints/popup/main.tsx

```diff
@@ -37,13 +39,15 @@ function HydrateAtoms({
 async function initApp() {
   const root = document.getElementById('root')!
   root.className = 'text-base antialiased w-[320px] bg-background'
-  const config = await storage.getItem<Config>('local:config')
+  const config = await storage.getItem<Config>('local:config') ?? DEFAULT_CONFIG
```

Comment:
acutally should use globalConfig here, which resolve the load and init issue for config